### PR TITLE
Admin email - add movie title and orgName 

### PR DESCRIPTION
**Pull request by KvnMrs**
issue: https://github.com/blockframes/blockframes/issues/8832

### CODE REVIEW

File: apps/backend-functions/src/movie.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ export async function onMovieUpdate(change: BlockframesChange<Movie>) {
     const movieWasSubmittedOn = wasSubmittedOn(before.app, after.app)[0];
     // Mail to supportEmails.[app]
     const from = getMailSender(movieWasSubmittedOn);
-    await sendMail(sendMovieSubmittedEmail(movieWasSubmittedOn, after), from, groupIds.noUnsubscribeLink);
+    await sendMail(await sendMovieSubmittedEmail(movieWasSubmittedOn, after, db), from, groupIds.noUnsubscribeLink);
```
Comment (phpgeekfr) [inline apps/backend-functions/src/movie.ts:143]: you already have the organizations here 
`const organizations = await getOrganizationsOfMovie(after.id);`
```suggestion
    await sendMail(sendMovieSubmittedEmail(movieWasSubmittedOn, after, organizations[0]), from, groupIds.noUnsubscribeLink);
```

File: apps/backend-functions/src/templates/mail.ts [hunk 2]
```diff
@@ -565,12 +566,15 @@ export function sendContactEmail(userName: string, userMail: string, subject: st
 }
 
 /** Send an email to supportEmails.[app](catalog & MF only) when a movie is submitted*/
-export function sendMovieSubmittedEmail(app: App, movie: Movie) {
+export async function sendMovieSubmittedEmail(app: App, movie: Movie, db: Firestore) {
+  const org = await db.collection('orgs').doc(movie.orgIds[0]).get()
+  const orgName = org.data().name;
+
```
Comment (phpgeekfr) [inline apps/backend-functions/src/templates/mail.ts:572]:
```suggestion
export function sendMovieSubmittedEmail(app: App, movie: Movie, org: Organization) {
  const orgName = org.name;

```

File: apps/backend-functions/src/movie.ts (hunk 1 above)
Comment (KvnMrs) [reply apps/backend-functions/src/movie.ts]: I  have to keep the 'await' without it i get an error, I find it litle strange

Comment (phpgeekfr) [reply apps/backend-functions/src/movie.ts]: remove async here `export async function sendMovieSubmittedEmai`

File: apps/backend-functions/src/templates/mail.ts [hunk 3]
```diff
@@ -33,6 +33,7 @@ import {
 } from '@blockframes/model';
 import { format } from 'date-fns';
 import { supportMailosaur } from '@blockframes/utils/constants';
+import { Firestore } from '@blockframes/firebase-utils';
```
Comment (phpgeekfr) [inline apps/backend-functions/src/templates/mail.ts:36]: you don't need this line

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**